src/wazuh_modules/syscollector/tests/sysCollectorFlatbuffers/syscollectorFb_test.h

```diff
@@ -0,0 +1,34 @@
+/*
+ * Wazuh SyscollectorImp
```

Comment:
Please, update the name of the file in the header.

Reply:
Done =)